python/lsst/validate/drp/matchreduce.py

```diff
@@ -409,3 +437,86 @@ def fullFilter(cat):
     # These attributes are not serialized
     blob.goodMatches = goodMatches
     blob.safeMatches = safeMatches
+
+
+def _loadPhotoCalib(butler, dataId, doApplyExternalPhotoCalib, externalPhotoCalibName):
+    """
+    Load a photoCalib object.
+
+    Parameters
+    ----------
+    butler: `lsst.daf.persistence.Butler`
+    dataId: Butler dataId
```

Comment:
Docstrings for these two (and below)? Is the "Butler dataId" a `dict` or an `lsst.daf.persistence.ButlerDataRef`?